### PR TITLE
DX-729 Added mCommerce request and response.

### PR DESCRIPTION
**Pull request by arebra**
After request from Mike Hellberg.

### CODE REVIEW

File: payments/swish/direct.md [hunk 1]
```diff
@@ -265,6 +264,88 @@ Content-Type: application/json
...
+            "operations": [
+                {
+                    "href": "swish://paymentrequest?token=LhXrK84MSpWU2RO09f8kUP-FHiBo-1pB",
+                    "method": "GET",
+                    "rel": "redirect-app-swish"
+                },
+                {
+                    "href": "https://api.payex.com/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f/sales/6bf31479-623f-418a-d69e-08d519f19722/qrcode.png",
+                    "method": "GET",
+                    "rel": "view-qr-code",
```
Comment (asbjornu) [inline payments/swish/direct.md]: The QR code stuff is not public yet, I think. Please remove.

File: payments/swish/direct.md [hunk 2]
```diff
@@ -265,6 +264,88 @@ Content-Type: application/json
...
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+{
+    "payment": "/psp/swish/payments/{{ page.payment_id }}",
+    "sale": {
+        "date": "23.10.2017 08:39:37 +00:00",
+        "paymentRequestToken": "97aabad01f3a4b4282c4e4fa36c8c259",
```
Comment (asbjornu) [inline payments/swish/direct.md]: I don’t think we should document this publicly.

Comment (NullableInt) [reply payments/swish/direct.md]: I agree on this.

**Comment by arebra** [comment]
Done the changes you asked for. Do we need tables for the requests? I know it's risky to assume that they should understand what the request contains, but... shouldn't they, though? 😄

**Comment by arebra** [comment]
Added request tables.

**Pull request merged**